pulpcore/tests/functional/api/test_upload.py

```diff
@@ -126,37 +124,35 @@ def test_upload_chunk_wrong_checksum(self):
 
     def test_upload_response(self):
         """Test upload responses when creating an upload and uploading chunks."""
-        self.client.response_handler = api.echo_handler
-
         upload_request = self.client.post(
             UPLOAD_PATH, {'size': self.size_file}
         )
 
         expected_keys = ['_href', '_created', 'size', 'completed']
 
-        self.assertEquals([*upload_request.json()], expected_keys)
+        self.assertEquals([*upload_request], expected_keys)
```

Comment:
```suggestion
        self.assertEqual([*upload_request], expected_keys, upload_request)
```


`assertEquals` is deprecated. See: https://docs.python.org/2/library/unittest.html#deprecated-aliases

Adding a message to the assertion in case of failure can be helpful in case of failure. 

See: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertEqual